ivy/functional/frontends/jax/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .numpy import mathematical_functions
+from ivy.functional.frontends.jax.numpy.mathematical_functions import *
```

Comment:
we should actually implement `jax.lax.tan` for the `jax` frontend. We are trying the approach of only implementing the `lax` frontend.

Reply:
sure...